nshlib/nsh_alias.c

```diff
@@ -47,22 +47,14 @@
 #define alias_head(list)        (FAR struct nsh_alias_s *)sq_peek(list)
 #define alias_remfirst(list)    (FAR struct nsh_alias_s *)sq_remfirst(list)
 
-/****************************************************************************
- * Private Types
- ****************************************************************************/
-
-/****************************************************************************
- * Private Data
- ****************************************************************************/
-
 /* Alias message format */
 
-static const char g_aliasfmt[]    = "alias %s='%s'\n";
-static const char g_savefailfmt[] = "alias %s='%s' failed\n";
+#define alias_format            "alias %s='%s'\n"
```

Comment:
should we keep g_ prefix like g_dd?

Reply:
Let's change to `atic const char *g_aliasfmt = "alias %s='%s'\n";` that will have basically the same effect with optimisation turned on

Reply:
Done

Reply:
So you still prefered to use define and not pointer to constant data? May I ask what is the difference in size/mem with define vs pointer to constant data?

Reply:
The problem is that the compiler will complain static symbols isn't used.